core-common/src/main/java/org/zalando/nakadi/annotations/validation/AnnotationKey.java

```diff
@@ -24,7 +24,7 @@
         message = "Key name should start and end with a letter or a digit")
 @Constraint(validatedBy = {})
 public @interface AnnotationKey {
-    String message() default "{org.zalando.nakadi.annotations.validation.AnnotationKey.message}}";
+    String message() default "{org.zalando.nakadi.annotations.validation.AnnotationKey.message}";
```

Comment:
Oh, wow — how did it work before? :)

Reply:
Apparently this code is never reached as message is overwrite by Pattern and NotEmpty annotations